libs/zephyr-rspress-plugin/src/zephyrRspressSSGPlugin.ts

```diff
@@ -0,0 +1,33 @@
+import type { RspressPlugin } from '@rspress/shared';
+import path from 'node:path';
+import { ZephyrEngine, ZephyrError, logFn, ze_log } from 'zephyr-agent';
+import { setupZeDeploy } from './internal/assets/setupZeDeploy';
+import { showFiles } from './internal/files/showFiles';
+import { walkFiles } from './internal/files/walkFiles';
+
+export const zephyrRspressSSGPlugin = ({ outDir = 'doc_build' }): RspressPlugin => {
+  const root = path.resolve(outDir);
+
+  const { zephyr_engine_defer, zephyr_defer_create } = ZephyrEngine.defer_create();
+  zephyr_defer_create({ builder: 'rspack', context: root });
+
+  return {
+    name: 'zephyr-rspress-plugin-ssg',
+    async afterBuild() {
+      try {
+        const files = await walkFiles(root);
+
+        if (files.length === 0) {
+          ze_log.package('No files found in output directory.');
```

Comment:
Change log type.
This one should be more related to uploading files.
Package relates to package.json parsin
```suggestion
          ze_log.upload('No files found in output directory.');
```